src/enzo/QuantumGetUnits.C

```diff
@@ -0,0 +1,144 @@
+/***********************************************************************
+/
```

Comment:
Could this header info be updated to reflect that this is a new type of get units? (and make sure that the math in the comments is correct)

Reply:
Actually, why do the `QuantumGet` functions exist? They appear to be the same as the standard `GetUnits` and `CosmologyGetUnits` (unless I missed something)

Reply:
QuantumGetUnits compute length and time units at a=1